parts/windows/kuberneteswindowssetup.ps1

```diff
@@ -215,7 +216,7 @@ try
 {
     Write-Log ".\CustomDataSetupScript.ps1 -MasterIP $MasterIP -KubeDnsServiceIp $KubeDnsServiceIp -MasterFQDNPrefix $MasterFQDNPrefix -Location $Location -AADClientId $AADClientId -NetworkAPIVersion $NetworkAPIVersion -TargetEnvironment $TargetEnvironment"
 
-    $WindowsCSEScriptsPackage = "aks-windows-cse-scripts-v0.0.16.zip"
+    $WindowsCSEScriptsPackage = "aks-windows-cse-scripts-v0.0.18.zip"

```

Comment:
Please also help to cache it in vhdbuilder/packer/generate-windows-vhd-configuration.ps1 and update the README in staging/cse/windows/README. NOTE: You do not need to update doc for v0.0.17 which contains some bug and will not be enabled before the fix in Windows.

https://github.com/Azure/AgentBaker/pull/2238/files#diff-1a71ca93f12803bce028e28b3600b14d149c348b3c829b2a7fe122b0e017fe25

Reply:
going to need to update this due to #2414

Reply:
Synced in teams